src/devices/bus/rs232/loopback.cpp

```diff
@@ -9,35 +9,68 @@ DEFINE_DEVICE_TYPE(RS232_LOOPBACK, rs232_loopback_device, "rs232_loopback", "RS2
 rs232_loopback_device::rs232_loopback_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock)
 	: device_t(mconfig, RS232_LOOPBACK, tag, owner, clock)
 	, device_rs232_port_interface(mconfig, *this)
+	, m_flow(*this, "FLOW_CONTROL")
 {
 }
 
+static INPUT_PORTS_START(rs232_loopback)
+	PORT_START("FLOW_CONTROL")
+	PORT_CONFNAME(0x01, 0x01, "Flow Control")
```

Comment:
Please attach in input changed callback on this so that flipping the switch while the emulation is running works.